docs/sphinx/using/backends/hardware.rst

```diff
@@ -5,6 +5,129 @@ CUDA-Q supports submission to a set of hardware providers.
 To submit to a hardware backend, you need an account with the respective provider.
 
 
+Infleqtion
```

Comment:
These sections are in alphabetical order.

Reply:
Apologies, reordered in https://github.com/NVIDIA/cuda-quantum/pull/2464/commits/75a0a08721ec16d202ef399e295c7ff82b7f9136